spring-cloud-kubernetes-fabric8-config/src/main/java/org/springframework/cloud/kubernetes/fabric8/config/LabeledSecretContextToSourceDataProvider.java

```diff
@@ -76,51 +67,30 @@ public Fabric8ContextToSourceData get() {
 
 			try {
 
-				LOG.info("Loading Secret(s) with labels '" + labels + "' in namespace '" + namespace + "'");
```

Comment:
moved things to Fabric8ConfigUtils... nothing broke here, we also have proper unit tests that would have failed otherwise